src/views/Home.vue

```diff
@@ -1,18 +1,61 @@
 <template>
-  <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <div>
+    <search @search:submit="search" class="search-field" />
+    <house-list
+      :houses="houses"
+      @house-list:load-page="loadPage"
+      :page="page"
+    />
   </div>
 </template>
 
 <script>
-// @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue';
+import HouseList from '@/components/HouseList.vue';
+import Search from '@/components/Search.vue';
+import API from '@/services/api';
+// import MOCK_API from '@/services/mockApi';
 
 export default {
   name: 'home',
+  data() {
+    return {
+      houses: [],
+      page: 1,
+      city: '',
+    };
+  },
+  async created() {
+    this.loadPage(this.page);
+  },
   components: {
-    HelloWorld,
+    'house-list': HouseList,
```

Comment:
Лишнее. В ПРе на тудушку уже писал про это. Можно просто 
```suggestion
    HouseList,
```

И так везде.